### DISCUSSION TITLE
Connection with Websphere Application Server

### DISCUSSION THREAD

**Discussion by praveen-kumar-mox** [arkmq-org/arkmq-org-broker-operator#537] [Q&A] [answered]
Hi Team,

We recently hosted Activemq Artemis in our EKS cluster. Now we want to connect it with Websphere application server hosted in different linux machine for send and receive messages. Could you provide the guideline on how we can connect Websphere application server with activemq ?
Could you also let us know from where we need to download the correct resource adapter with name and version?

**Comment by brusdev** [comment, marked as answer]
@praveen-kumar-mox this is a question for the users mailing list of the Apache ActiveMQ Artemis project: users@activemq.apache.org
